src/chrome/content/rules/Stremio.io.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Stremio.io">
+        <target host="stremio.cz" />
```

Comment:
stremio.cz doesn't exist, stremio.io is parked. Did you mean to target strem.io?